### PR TITLE
make sure the python really did its job

### PR DESCRIPTION
**Pull request by dothebart**

### CODE REVIEW

File: scripts/runFullTests.fish [hunk 1]
```diff
@@ -16,8 +16,12 @@ end
 function launchSingleTests
   echo "Using test definitions from arangodb repo"
   python3 "$WORKSPACE/jenkins/helper/test_launch_controller.py" "$INNERWORKDIR/ArangoDB/tests/test-definitions.txt" -f launch --full "$ENTERPRISE_ARG"
-  and set -xg result "GOOD"
-  or set -xg result "BAD"
+  if test "$status" == 0 and test -f work/testRuns.html
+    set -xg result "GOOD"
+  else
+    set -xg result "BAD"
+    echo "python exited $status"
```
Comment (KVS85) [inline scripts/runFullTests.fish:23]: `$status` isn't persisted.

File: scripts/runFullTests.fish [hunk 2]
```diff
@@ -26,8 +30,12 @@ end
 
 function launchGTest
   python3 "$WORKSPACE/jenkins/helper/test_launch_controller.py" "$INNERWORKDIR/ArangoDB/tests/test-definitions.txt" -f launch --gtest "$ENTERPRISE_ARG"
-  and set -xg result "GOOD"
-  or set -xg result "BAD"
+  if test "$status" == 0 and test -f work/testRuns.html
+    set -xg result "GOOD"
+  else
+    set -xg result "BAD"
+    echo "python exited $status"
```
Comment (KVS85) [inline scripts/runFullTests.fish:37]: `$status` isn't persisted.

File: scripts/runFullTests.fish [hunk 3]
```diff
@@ -37,8 +45,12 @@ end
 function launchClusterTests
   echo "Using test definitions from arangodb repo"
   python3 "$WORKSPACE/jenkins/helper/test_launch_controller.py" "$INNERWORKDIR/ArangoDB/tests/test-definitions.txt" -f launch --cluster --full "$ENTERPRISE_ARG"
-  and set -xg result "GOOD"
-  or set -xg result "BAD"
+  if test "$status" == 0 and test -f work/testRuns.html
+    set -xg result "GOOD"
+  else
+    set -xg result "BAD"
+    echo "python exited $status"
```
Comment (KVS85) [inline scripts/runFullTests.fish:52]: `$status` isn't persisted.

File: scripts/runFullTests.fish [hunk 4]
```diff
@@ -48,8 +60,12 @@ end
 function launchSingleClusterTests
   echo "Using test definitions from arangodb repo"
   python3 "$WORKSPACE/jenkins/helper/test_launch_controller.py" "$INNERWORKDIR/ArangoDB/tests/test-definitions.txt" -f launch --single_cluster --full "$ENTERPRISE_ARG"
-  and set -xg result "GOOD"
-  or set -xg result "BAD"
+  if test "$status" == 0 and test -f work/testRuns.html
+    set -xg result "GOOD"
+  else
+    set -xg result "BAD"
+    echo "python exited $status"
```
Comment (KVS85) [inline scripts/runFullTests.fish:67]: `$status` isn't persisted.

**Review by KVS85 — CHANGES_REQUESTED** [review]

**Comment by dothebart** [comment]
https://jenkins01.arangodb.biz/view/PR/job/arangodb-matrix-pr/30238/ launched with [b83deca](https://github.com/arangodb/oskar/pull/519/commits/b83decae4618cfc23c77170a993a99232a949706)
https://jenkins01.arangodb.biz/job/arangodb-matrix-pr-mac-onlycatch.x86-64/25290/ launched with [4ea277c](https://github.com/arangodb/oskar/pull/519/commits/4ea277cd734cb9ec4f3b9d4183ca0116504ac506) to revalidate the forgotten condition.

**Review by naushniki — APPROVED** [review]
LGTM

**Review by KVS85 — APPROVED** [review]

**Pull request merged**